mne/transforms.py

```diff
@@ -690,7 +690,19 @@ def _cart_to_sph(cart):
 
 
 def _sph_to_cart(sph):
```

Comment:
```suggestion
def _sph_to_cart(sph_pts):
```

if you want to match docstring.

you'll need to update what's new